assemble/src/main/scripts/generate-download-script.sh

```diff
@@ -0,0 +1,56 @@
+#! /usr/bin/env bash
+
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script will generate a DEPENDENCIES listing of packaged dependencies
+
+in=target/dependencies.raw.txt
+out=target/download-dependencies
+
+cat >"$out" <<'EOF'
+#! /usr/bin/env bash
+# This script downloads the following jars, identified by their maven
+# coordinates, using the maven-dependency-plugin.
+#
+# DISCLAIMER: This is only one possible way to download a set of dependencies
```

Comment:
> the upstream Accumulo project cannot take responsibility for the dependencies

But why? I feel like this is missing from the argument. There are some transitive dependencies which we need to interact with HDFS and YARN that (I assume) you're pointing at, but that isn't all of our dependencies.

We can't get around needing certain ones for HDFS, YARN and ZK which I agree with you on. However, we still have a known set of dependencies that Accumulo itself needs. We as a community _should_ be aware of security concerns and make sure that we tell users "hey, you should update to a newer version because of some bug or CVE". Furthermore, we don't need to be shackled to the dependencies that HDFS et. al require. We can relocate the dependencies _we use_ and reduce the effort that is required to deploy Accumulo.